command/daemon/daemon.go

```diff
@@ -101,59 +100,55 @@ func (c *daemonCommand) run(*kingpin.ParseContext) error {
 		),
 	)
 	// generate le cert files if needed
-	err = le.GenerateLECerts(config.Runner.Name, config.DefaultPoolSettings.CertificateFolder)
+	err = le.GenerateLECerts(env.Runner.Name, env.DefaultPoolSettings.CertificateFolder)
```

Comment:
separate pr, but we need to generate unique certs per-pipeline for security reasons.  Also could rename package to `certs` and function to `certs.Generate` for more idiomatic naming.